### ISSUE TITLE
WellLogViewer, Typescript - Sideways scrolling not possible in Firefox

### ISSUE THREAD

**Issue by Anders2303** [equinor/webviz-subsurface-components#2246]
The viewer's scrollbars are not visible in Firefox. I think this is just a visual issue with how Firefox hides scrollbars until you hover over the scrollable element. Is the overlay element or something else blocking this hover from being visible? As far as I know, this should be a simple fix; you should be able to apply CSS to make sure the scrollbar is always visible,

Furthermore, you **should** ([atleast according to some guidelines](https://learn.microsoft.com/en-us/windows/apps/design/controls/scroll-controls?tabs=scrollview)) be able to scroll sideways using `Shift+Scroll-wheel`, but viewer does not support scrolling modifiers.

**Comment by hkfb** [comment]
> The viewer's scrollbars are not visible in Firefox. I think this is just a visual issue with how Firefox hides scrollbars until you hover over the scrollable element. Is the overlay element or something else blocking this hover from being visible? As far as I know, this should be a simple fix; you should be able to apply CSS to make sure the scrollbar is always visible,

Good catch. Maybe the component should have a `style` prop? Although we have to think about how that interacts with the `template` prop.
